src/mainboard/opentitan/crb/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "crb"
+version = "0.1.0"
+authors = ["Oreboot Authors"]
+edition = "2018"
+build = "build.rs"
+
+[dependencies]
+arch = { path = "../../../arch/riscv/rv32"}
```

Comment:
should be soc, not arch.

Reply:
We need to import the arch to call "arch::halt()".

Reply:
has to be a better way. Just has to.